StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+            IWebElement InvulVelden = Driver.FindElement(By.ClassName("iframeDialog"));
+            InvulVelden.FindElement(By.Name("AddressLine1")).SendKeys("Danzigerkade 165");
+            InvulVelden.FindElement(By.Name("City")).SendKeys("Amsterdam");
+            InvulVelden.FindElement(By.Name("PostalCode")).SendKeys("1013 AP");
+            InvulVelden.FindElement(By.Name("FindButton")).Click();
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("subcol5")).Displayed);
+            IWebElement LocationSelector = Driver.FindElements(By.ClassName("subcol5")).ElementAt(0);
+            LocationSelector.FindElement(By.Name("SelectLocation")).Click();
+            Driver.SwitchTo().DefaultContent();
+            System.Threading.Thread.Sleep(1000);
+        }
+         
+        [When(@"I proceed to the payment page")]
+        public void WhenIProceedToThePaymentPage()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.Id("pickUpPointForm")).Displayed);
+            ((IJavaScriptExecutor)Driver).ExecuteScript("arguments[0].scrollIntoView(true);", Driver.FindElement(By.Id("pickUpPointForm")));
+            IWebElement persDetails = Driver.FindElement(By.Id("pickUpPointForm"));
+            IWebElement TitleSelector = persDetails.FindElement(By.Id("pickUpPoint_title_chosen"));
+            TitleSelector.Click();
+            IWebElement TitleSelected = TitleSelector.FindElement(By.ClassName("chosen-drop"));
+            TitleSelected.FindElements(By.ClassName("active-result")).ElementAt(1).Click();
+            //var dropdownpers = Driver.FindElements(By.ClassName("active-result")).Where(x => x.Displayed);
+            //dropdownpers.ElementAt(1).Click();
+            IWebElement Voornaam = Driver.FindElement(By.ClassName("pupFirstName"));
+            Voornaam.FindElement(By.Name("firstName")).SendKeys("Theo");
+            IWebElement Achternaam = Driver.FindElement(By.ClassName("pupLastName"));
+            Achternaam.FindElement(By.Name("lastName")).SendKeys("Test");
+            IWebElement Telefoon = Driver.FindElement(By.ClassName("pupPhone"));
+            Telefoon.FindElement(By.Name("phone1")).SendKeys("0612345678");
+            IWebElement Email = Driver.FindElement(By.ClassName("pupEmail"));
+            Email.FindElement(By.Name("email1")).SendKeys("test@test.test");
+            IWebElement Betalen = Driver.FindElement(By.ClassName("checkoutControls"));
+            Betalen.FindElement(By.Id("continueToPayment")).Click();
+        }
+
+        [When(@"I click the back button in my browser")]
+        public void WhenIClickTheBackButtonInMyBrowser()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("paymentOptionSelect")).Displayed);
+            Driver.Navigate().Back();
+        }
+         
+        [When(@"I change the delivery method to standard")]
+        public void WhenIChangeTheDeliveryMethodToStandard()
```

Comment:
We should have a general function to change delivery method to any options, so it should accept parameter as shipping method name, and within function, you could based on the parameter,  to do different things